packages/components/src/styles/components/dropdown.scss

```
@@ -73,16 +74,26 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
   }
 }
 
+// Icon sizes
+
 .hds-dropdown-toggle-icon--size-small {
   .hds-dropdown-toggle-icon__wrapper {
-    width: 24px; // same as the `button[small]` minus 2*2px becaus there are a 1px padding and 1px border to take into account
+    width: 24px; // same as the `button[small]` minus 2*2px because there are a 1px padding and 1px border to take into account
```

I'm the one who wrote the comment (I think) and it's not clear even to me 😀
```suggestion
    width: 24px; // we use this element to provide the right size for the parent "button" so it matches the `button[small]`
```
(feel free to edit/reword it)